app/models/user.rb

```diff
@@ -7,4 +7,15 @@ class User < ApplicationRecord
   has_many :collections
   has_many :manifests, -> { distinct }, through: :collections # the User's library
   has_many :annotations
+
+  after_create :create_sample_data
+
+  def create_sample_data
+    samples = JSON.parse(File.read('config/sample_data.json'))
```

Comment:
The reasoning was allowing the class to add sample data from anywhere. But encapsulating everything might make more sense.